src/mots/cli.py

```diff
@@ -302,4 +251,35 @@ def create_parser():
     )
     export_parser.set_defaults(func=export)
 
+    # Add path argument for all parsers that need it.
+    for _parser in (
+        init_parser,
+        module_parser,
+        validate_parser,
+        clean_parser,
+        check_hashes_parser,
+        query_parser,
+        export_parser,
+    ):
+        _parser.add_argument(
+            "--path",
+            "-p",
+            type=Path,
+            help="the path of the repo config file",
+            default=settings.DEFAULT_CONFIG_FILEPATH,
+        )
+
+    # Add repo-path argument for all parsers that need it.
+    for _parser in (
+        validate_parser,
+        clean_parser,
+    ):
+        _parser.add_argument(
+            "--repo-path",
+            "-r",
+            type=str,
```

Comment:
Now that you mention this, I am rethinking the need for this argument altogether, since we can always deduce it from the config file path (with the assumption that the config file is always in the repo root).